public/data/zougari47.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Ahmed Zougari",
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Frontend Developer | JavaScript 💛",
+  "avatar": "https://github.com/zougari47.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/zougari47",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zougari47",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i`; it will look better, IMO. What do you think? :)
```suggestion
      "name": "Follow me on LinkedIn",
```